ChatQnA/docker_compose/intel/hpu/gaudi/compose.yaml

```diff
@@ -65,20 +63,23 @@ services:
       HUGGINGFACEHUB_API_TOKEN: ${HUGGINGFACEHUB_API_TOKEN}
     restart: unless-stopped
   tei-reranking-service:
-    image: ghcr.io/huggingface/text-embeddings-inference:cpu-1.5
+    image: ghcr.io/huggingface/tei-gaudi:latest
     container_name: tei-reranking-gaudi-server
     ports:
       - "8808:80"
     volumes:
       - "./data:/data"
-    shm_size: 1g
+    runtime: habana
+    cap_add:
+      - SYS_NICE
+    ipc: host
     environment:
       no_proxy: ${no_proxy}
       http_proxy: ${http_proxy}
       https_proxy: ${https_proxy}
-      HUGGINGFACEHUB_API_TOKEN: ${HUGGINGFACEHUB_API_TOKEN}
```

Comment:
Why remove HUGGINGFACEHUB_API_TOKEN

Reply:
HF_HUB_DISABLE_PROGRESS_BARS, HF_HUB_ENABLE_HF_TRANSFER can also be safe to keep.

Reply:
Gaudi TEI doesn't need token for embedding and reranking model.